feat: Add ignoreCommits config value to skip single commit linting when unnecessary (#18)
CondeNast/conventional-pull-request-action: README.md

          commitlintRulesPath: "./commitlint.rules.js" # default: undefined
          # if the PR contains a single commit, fail if the commit message and the PR title do not match
          commitTitleMatch: "true" # default: 'true'

>           # if you squash merge PRs and have your github repo set to always use the PR title as the merge commit, you can disable all linting of commits

Reviewer: It'll help if this uses the exact same language as the Github UI (at least in English):
```suggestion
          # if you squash merge PRs and enabled "Default to PR title for squash merge commits", you can disable all linting of commits
```
